packages.txt

```diff
@@ -1172,30 +1358,33 @@ tcl8.5						install
 tcpd						install
 telnet						install
 tig						install
-time						install
+tk8.6-blt2.5					install
 tmux						install
 toilet						install
 toilet-fonts					install
 topgit						install
 tor						install
 tor-arm						install
+tor-geoipdb					install
```

Comment:
there are tons of changes in the packages.txt, i really don't know why `apt install nginx` changed this file so much